interfaces/policy/policy.go

```diff
@@ -30,6 +30,50 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+// InstallCandidateMinimalCheck represents a candidate snap installed with --dangerous flag that should pass minimum checks
```

Comment:
could we put this either completely at end or at least after InstallCandidate stuff, is a corner case, not the normal case?

Reply:
(A different approach, but probably not clearer would have been to reuse InstallCandidate but add a set of CheckMinimal and its *Minimal helper methods. Just as a note, don't think it's clearer)